tests/Crc32ContentHashTest.php

```diff
@@ -18,9 +18,15 @@ class Crc32ContentHashTest extends TestCase
 {
     public function testHashMustReturnTheContentDisgestWithCRC32()
     {
+        $filepath = __DIR__ . '/test.txt';
```

Comment:
What do you think about using the temporal returned by the function `sys_get_temp_dir`? this way you do not need to add `test.txt` to `.gitignore` file. Same situation for the file `tests/Crc32MetaDataHashTest.php`